This is internal patch implementing mutt's issue:
http://dev.mutt.org/trac/ticket/3392 

This link is no longer available as Mutt's issue ticketing system moved 
since the time of creating this issue from http://dev.mutt.org/hg/mutt/
to https://gitlab.com/muttmua/mutt

However, there is still at least some external discussion found at:
https://www.mail-archive.com/mutt-users@mutt.org/msg41591.html

diff --git a/pager.c b/pager.c
index 1d67971..b0a5da3 100644
--- a/pager.c
+++ b/pager.c
@@ -1950,6 +1950,8 @@ mutt_pager (const char *banner, const char *fname, int flags, pager_t *extra)
   }
   unlink (fname);
 
+  clearok (stdscr, TRUE);
+
   /* Initialize variables */
 
   if (IsHeader (extra) && !extra->hdr->read)
